plugins/notifications-backend/migrations/20240313_broadcast.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+exports.up = async function up(knex) {
+  await knex.schema.createTable('broadcast', table => {
+    table.uuid('id').primary();
+    table.string('title').notNullable();
+    table.text('description').nullable();
+    table.string('severity', 8).notNullable();
```

Comment:
Might become numeric within https://github.com/backstage/backstage/pull/23371.

```table.tinyint('severity').notNullable()```

Reply:
Yeah, lots of conflicts incoming.